### PR TITLE
test(atlas): atlas connectivity tests

### PR DESCRIPTION
**Pull request by daprahamian**
Fixes NODE-1221

### CODE REVIEW

File: atlas.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    .then(() => topologyIsMaster(topology))
+    .then(() => topologyFindCmd(topology))
+    .then(() => console.log(`${config.name} passed`))
+    .catch(e => {
+      console.log(`${config.name} failed`);
+      throw e;
+    });
+}
+
+configs
```
Comment (mbroadst) [inline atlas.js]: is there a particular reason you chose not to use mocha for this?

Comment (daprahamian) [reply atlas.js]: a) it seemed like overkill
b) I explicitly wanted to not have a possibility of an error message leaking out so as to not expose the url

Comment (mbroadst) [reply atlas.js]: My concern is that this special cases a class of test we have, how is "atlas connect" different than "ssl" or "auth"?

Comment (daprahamian) [reply atlas.js]: a. It's different in that we don't run `ssl` or `auth` anymore ;)
b. The explicit requirements that we not expose the atlas urls has me a little paranoid.

I kind of view this specific test case as similar to driverBench.

Comment (mbroadst) [reply atlas.js]: 👍

**Pull request merged**